web/elm/tests/TopBarTests.elm

```diff
@@ -1639,25 +1519,390 @@ all =
                        ]
                     ++ [ context "on ESC keypress"
                             (Tuple.first
-                                >> Application.update (Msgs.DeliveryReceived <| KeyDown 27)
+                                >> Application.update
+                                    (ApplicationMsgs.DeliveryReceived <|
+                                        KeyDown 27
+                                    )
                             )
                             [ it "search input is blurred" <|
                                 Tuple.second
-                                    >> Expect.equal [ ( Effects.SubPage 1, "", Effects.Blur TopBar.searchInputId ) ]
+                                    >> Expect.equal [ Effects.Blur SearchBar.searchInputId ]
                             ]
                        ]
                 )
             , context "after receiving FocusMsg and then BlurMsg"
-                (Application.update (Msgs.SubMsg 1 <| SubPage.Msgs.DashboardMsg <| Dashboard.Msgs.FromTopBar <| TopBar.Msgs.FocusMsg)
+                (Application.update (ApplicationMsgs.Update Msgs.FocusMsg)
                     >> Tuple.first
-                    >> Application.update (Msgs.SubMsg 1 <| SubPage.Msgs.DashboardMsg <| Dashboard.Msgs.FromTopBar <| TopBar.Msgs.BlurMsg)
+                    >> Application.update
+                        (ApplicationMsgs.Update Msgs.BlurMsg)
                     >> viewNormally
                 )
                 [ it "hides the dropdown" <|
                     Query.findAll [ id "search-dropdown" ]
                         >> Query.count (Expect.equal 0)
                 ]
             ]
+        , describe "pause toggle" <|
```

Comment:
Ahhhh, here it is. I was getting worried when I didn't see it in pipelineTests. ;)